src/Test/TestAIMuriel.java

```diff
@@ -144,14 +183,14 @@ private boolean testClearAttackRoute()
     // Where are things?
     XYCoord facPos = new XYCoord(7, 8);
 
-    // Add an enemy tank on a neutral fac, flanked by friendly infs, with a friendly tank nearby.
-    Unit nmeTank = addUnit(testMap, testCo2, "Tank", facPos);
-    Unit myTank = addUnit(testMap, testCo1, "Md Tank", facPos.up().left());
+    // Add an enemy copter on a neutral fac, flanked by friendly infs, with a friendly AA nearby.
+    Unit nmeCopter = addUnit(testMap, testCo2, "B-Copter", facPos);
```

Comment:
Why change the units around?

Reply:
To change the value proposition, since "judge this a good attack to make" doesn't seem to be a core part of the test (if I thought it was, I'd have thrown it out of scope for Wally).
Wally only wants to go aggro if he can deal about as much damage as he'd take by losing the unit... and this new unit pair satisfies that.

He should, at some point, figure out that you shouldn't count the unit you're about to first-strike as a significant threat to counterattack a play, but that's more AI juice than I think I want to delay the next build for.